modules/javafx.graphics/src/main/java/com/sun/scenario/animation/AbstractPrimaryTimer.java

```diff
@@ -79,13 +86,15 @@ public abstract class AbstractPrimaryTimer {
         return null;
     };
 
-    private PulseReceiver receivers[] = new PulseReceiver[2];
+    @SuppressWarnings("unchecked")
+    private ReceiverRecord<PulseReceiver>[] receivers = new ReceiverRecord[2];
```

Comment:
weird that there is no way to do it cleanly without `@SuppressWarnings("unchecked")`

BTW, Eclipse shows 5,608 hits if I enable "unchecked generic type operation" warning.  Pretty useless, if you ask me.